src/index.js

```diff
@@ -1 +1 @@
-export { default as default } from "./RangeSlider.svelte";
\ No newline at end of file
+export { default } from "./RangeSlider.svelte";
```

Comment:
while I see this seems to be correct, I have two comments;
1) it's unrelated to the issue
2) this component is cloned from the official component template; https://github.com/sveltejs/component-template/blob/master/src/index.js which uses `{ default as default }` which makes me wonder why they did that?

Reply:
Unrelated indeed. Happy to leave it as is or submit a separate PR if you prefer.

I can only assume whoever wrote the component template didn't know the short version is allowed and that the two are equivalent. I can vouch for it working though. I use it in all my own packages. See e.g. https://github.com/janosh/svelte-multiselect/blob/main/src/lib/index.ts.